openlibrary/macros/EditionNavBar.html

```diff
@@ -1,4 +1,9 @@
-$def with (edition_count=1, show_observations=False)
+$def with (reader_observations, edition_count=1, show_observations=False)
+
+$if reader_observations['total_respondents']:
+  $ total_reviews = reader_observations['total_respondents']
+$else:
+  $ total_reviews = ''
```

Comment:
```suggestion
$ total_reviews = reader_observations.get('total_respondents', '')
```
This will try to access `total_respondents`, and return an empty string if the key is not found.

Reply:
Ah nice! Yeah this is much cleaner